src/app/api/Relic/relic.ts

```diff
@@ -0,0 +1,25 @@
+import requests from '../requests';
+
+const RelicAPI = {
+    updateRelicFile: (relicId: number) =>
+        requests.put(`api/relics/file/update/${relicId}`, relicId),
+    editRelic: (relicId: number) =>
+        requests.put(`api/relics/edit/${relicId}`, relicId),
+    //  api/relics/filter (should i create enum ?)
+    uploadRelicFile: (relicId: number, file: string) =>
+        requests.post(`api/relics/edit/${relicId}`, file),
+    // api/relics/create (too many fields for today)
+    // [POST] api/relics/change-favorite accepts no params??
```

Comment:
currently it is selected by random. Can be modified to both either receive a param or to be random